packages/design-system/src/components/checkbox/index.js

```diff
@@ -95,7 +95,7 @@ const BaseCheckbox = ({ checked, disabled, className = '', ...props }, ref) => (
     <input
       type="checkbox"
       ref={ref}
-      checked={checked}
+      checked={Boolean(checked)}
```

Comment:
🤔 Do we need this? Having the prop type of `checked: PropTypes.bool` should be enough to warn us.

But I get that we could still get past that pretty easily.

Reply:
There were lots of prop types warnings in the karma tests that nobody paid attention to, so 🤷‍♂️